fortes_webservice_wrapper/wrapper/soap.py

```diff
@@ -0,0 +1,11 @@
+from zeep import Client
+
+from ..constants import WSDL_URL
+
+
+class BaseSoapWrapper:
+    def __init__(self):
+        self.wsdl = WSDL_URL
```

Comment:
Com isso você deixa o usuário da LIB passar o argumento no init do wrapper também! Não obrigando ele a utilizar o ENV!

```suggestion
    def __init__(self, wsdl_url=None):
        if wsdl_url is None:
            self.wsdl = WSDL_URL
        else:
            self.wsdl = wsdl_url
        
```